types/crypto-random-string/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for crypto-random-string 1.0
+// Project: https://github.com/sindresorhus/crypto-random-string
+// Definitions by: Manuel Warum <https://github.com/MrManny>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Generate a hexadecimal string of the given length.
+ * @param length the length of the random string.
+ * @return a string of the specified length.
+ */
+declare function cryptoRandomString(length: number): string;
+
+export default cryptoRandomString;
```

Comment:
Since this package exports by modules.exports = you want to write it as . See https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default